DCA/README.md

```diff
@@ -0,0 +1,25 @@
+# DCA
+This folder contains adapter and Dockerfile for [DCA](https://github.com/Luffy03/DCA) algorithm
+
+## Building Docker Image
+1. Download weights from [here](https://drive.google.com/drive/folders/1oenWpYADqd-tTx7JeDQknxRNd3mgW2kQ) and put them into ./weights folder
```

Comment:
This must be put into docker, all dependencies must be in docker. See [example](https://github.com/prime-slam/line-detection-association-dockers/blob/main/detectors/LETR/Dockerfile) of model download in docker